draft-ietf-lamps-cms-ml-dsa.md

```diff
@@ -193,17 +185,27 @@ ML-DSA-PublicKey ::= OCTET STRING
 ML-DSA-PrivateKey ::= OCTET STRING
 ~~~
 
+Algorithm 22 in Section 7.2 of {{FIPS204}} defines the raw octet string encoding of an ML-DSA public key.
+When used in a SubjectPublicKeyInfo type, the subjectPublicKey BIT STRING contains the raw octet string encoding of the public key.
```

Comment:
"octet string" versus "OCTET STRING" might be a bit confusing - FIPS 204 describes the output of Algorithm 22 as a byte string, so that could be an alternative. Alternatively, something like "the subjectPublicKey BIT STRING contains the raw octet string encoding of the public key, i.e. without any additional ASN.1 wrapping". I do think it makes sense as currently written, it just took me a second read to check that the lack of capital letters was deliberate.